pkg/kapp/cmd/app/delete.go

```diff
@@ -46,7 +46,8 @@ func NewDeleteCmd(o *DeleteOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Co
 		Short: "Delete app",
 		RunE:  func(_ *cobra.Command, _ []string) error { return o.Run() },
 		Annotations: map[string]string{
-			cmdcore.AppHelpGroup.Key: cmdcore.AppHelpGroup.Value,
+			cmdcore.AppHelpGroup.Key:        cmdcore.AppHelpGroup.Value,
+			"cli.carvel.dev/tty-by-default": "",
```

Comment:
can we pull this into a constant somewhere. seems like going to be easy to "typo" it and not realize it.

Reply:
Yep, I thought of moving it to cmdcore, but it felt weird to have it there and I kept it as it is. I will move it to cmdapp as it is being used there the most and it won't cause any cyclic dependency as well.

Reply:
Done.